src/PlatformSwitch/PlatformSwitch.tsx

```diff
@@ -159,20 +168,19 @@ export const PlatformSwitch: FC<PlatformSwitchProps> = ({
   ));
 
   if (isMobile) {
-    const SelectedIcon = PlatformIcons[selected || 'Builder'];
+    const SelectedIcon = PlatformIcons[selected || Platform.Builder];
     return (
-      <Box sx={{ position: 'relative' }}>
+      <Box sx={{ position: 'relative', height: '40px' }}>
```

Comment:
I am not sure why this is needed, maybe you can get rid of the box completely and add the `sx={{ position: "relative" }}` to the SpeedDial directly ?

Reply:
@LinaYahya I see that without it the header takes the height of the speeddial, I think it is ok to keep it. maybe try to remove the box and see if adding the relative position on the speeddial directly can fix it, otherwise this is fine.

Reply:
Unfortunately, it didn't work as the speed dial has an absolut position, so it should be wrapped within the relative container